Speakers.UI/src/components/SpeakingEngagementsList/SpeakingEngagementsList.js

```
@@ -1,11 +1,63 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { getSpeakingEngagements } from '../../api/speakingEngagements';
+import SpeakingEngagement from './SpeakingEngagement';
+import Album from '../materialComponents/album';
+
+
+export const loadEngagements = (engagements, setEngagements) => () => {
+  (async () => {
+    setEngagements({
+      ...engagements,
+      loading: true,
+    });
+
+    try {
+      // eslint-disable-next-line no-console
+      const engagementsIn = await getSpeakingEngagements().catch((error) => console.log(error));
```

does catching the promise eat the error too?

---

The getSpeakingEngagements() function has get().then() in it, so the catch goes on top of the .then() call.